.github/workflows/deploy.yml

```
@@ -0,0 +1,16 @@
+name: Deploy
+on:
+  push:
+    tags:
+    - '*'
+  workflow_run:
+    workflows: ["Package"]
```

```suggestion
    workflows: 
      - "Package"
```
というスタイルのほうが統一感があって良さげです。

---

fix: [6dc2e44](https://github.com/groonga/groonga/pull/2156/commits/6dc2e449a7cf634923e4d024f519a7c6ebfd3c65) ありがとうございます！